bccsp/pkcs11/pkcs11_test.go

```diff
@@ -218,8 +214,11 @@ func TestPKCS11ECKeySignVerify(t *testing.T) {
 	}
 
 	msg1 := []byte("This is my very authentic message")
-	msg2 := []byte("This is my very unauthentic message")
 	hash1, _ := currentBCCSP.Hash(msg1, &bccsp.SHAOpts{})
+	/**
+	msg2 := []byte("This is my very unauthentic message")
+
+	/**
```

Comment:
?

Reply:
I forgot I had made these adjustments on this local branch.  Have a push coming